src/Telemetry/Microsoft.DotNet.AzureDevOpsTimelineTests/TimelineBuilder.cs

```diff
@@ -77,5 +77,31 @@ public TimelineBuilder AddRecord(string previousAttemptTimelineId)
 
             return this;
         }
+
+        public TimelineBuilder AddRecord(string workerName, string recordName, string logUrl)
+        {
+            int nextId = 1;
+            if (records.Any())
+            {
+                nextId += records.Max(r => int.Parse(r.Id));
+            }
+
+            TimelineRecord record = new TimelineRecord()
```

Comment:
Should the image name be here as well or that is a different record?

Reply:
Image name is a property of the AugmentedTimelineRecords class, which is a wrapper around this one